### ISSUE TITLE
Dead or Alive Mame roms error 2

### ISSUE THREAD

**Issue by legluondunet** [libretro/flycast#501]
Hello,
just tried this Mame roms:

Dead or Alive 2 | doa2 |   
Dead or Alive 2 (Rev A) | doa2a |   
Dead or Alive 2 Millennium | doa2m

They don't work, I have only a NAOMI logo with this message:

![Capture d’écran de 2019-03-10 17-40-53](https://user-images.githubusercontent.com/325405/54088205-d2d88180-435b-11e9-8520-886aa203c13a.png)

Someone has better luck?

_Originally posted by @legluondunet in https://github.com/libretro/reicast-emulator/issues/136#issuecomment-471322659_

**Comment by legluondunet** [comment]
I tried to change country without success.

**Comment by barbudreadmon** [comment]
@legluondunet make sure you don't have a `naomi_boot.bin` in your bios folder.

**Comment by legluondunet** [comment]
I had one, I erased it. But I have still the error message.

**Comment by legluondunet** [comment]
I looked in the core information, I have all the files, optional too, all are validated by RA.

**Comment by legluondunet** [comment]
I joined you the log file 
[retroarch.log](https://github.com/libretro/reicast-emulator/files/2955503/retroarch.log)

**Comment by legluondunet** [comment]
It works now, thank you.

**Comment by legluondunet** [comment]
I had this file, I erased it but I had still the error message. Then I changed region to "USA" instead "Default" and now 
Dead or Alive 2 | doa2 |  
Dead or Alive 2 Millennium | doa2m
Dead or Alive 2 (Rev A) | doa2a | 
loaded correctly.  
Could it be automatic?

**Comment by barbudreadmon** [comment]
@legluondunet yes, i'll do the change in the source code to force japan bios (or whatever region it needs) for those roms, i had a guess it would be related to this : your `naomi_boot.bin` was probably from an incompatible region and was taking priority over your `naomi.zip`, ignoring the setting in core options

@flyinghead don't you think we should remove support for deprecated `naomi_boot.bin` ? Keeping it is causing more issues than it solves.

**Comment by legluondunet** [comment]
I tested the 3 "Dead or ALive" Mame roms with all region: USA, Japan or Europe.
They launched correctly with all three region, I obtained error 2 message only if region set to default.
Other thing, each time I launch the game I obtain this loading screen:

![Capture d’écran de 2019-03-12 08-23-44](https://user-images.githubusercontent.com/325405/54182439-d2e29980-44a1-11e9-8f08-09cff6f5f954.png)

![Capture d’écran de 2019-03-12 08-23-58](https://user-images.githubusercontent.com/325405/54182437-d24a0300-44a1-11e9-9a09-9b5d740067ff.png)

then the game launched normally.
